tests/data/xchacha20-poly1305-ietf_ref.txt

```diff
@@ -0,0 +1,8 @@
+# Test vector from libsodium's xchacha20poly1305-ietf test
```

Comment:
I am having hard time finding these test vectors from libsodium and/or RFC7539. Could you please add a link that allows the verification of these values.